src/main/resources/db/changelog/changes/05-add-users.yml

```diff
@@ -0,0 +1,47 @@
+databaseChangeLog:
```

Comment:
I'd rename this file to `05-insert-users` to avoid confusion with `add-users-table`, also check other scripts and rename them accordingly